### PR TITLE
Code clean-up

### PR DESCRIPTION
**Pull request by elenadoty**
Removes unused code.

#no-changelog

### CODE REVIEW

File: Crashlytics/Crashlytics/Handlers/FIRCLSMachException.c [hunk 1]
```diff
@@ -287,12 +253,6 @@ static bool FIRCLSMachExceptionRegister(FIRCLSMachExceptionReadContext* context,
     return false;
   }
 
-  // Get the desired mask, which covers all the mach exceptions we are capable of handling,
-  // but clear out any that are in our ignore list.  We do this by ANDing with the bitwise
-  // negation.  Because we are only clearing bits, there's no way to set an incorrect mask
-  // using ignoreMask.
-  context->mask = FIRCLSMachExceptionMask() & ~ignoreMask;
```
Comment (samedson) [inline Crashlytics/Crashlytics/Handlers/FIRCLSMachException.c:294]: Is this property still in place on FIRCLSMachExceptionReadContext? We might need to set this

Comment (elenadoty) [reply Crashlytics/Crashlytics/Handlers/FIRCLSMachException.c:294]: Done

File: Crashlytics/Crashlytics/Handlers/FIRCLSMachException.c [hunk 2]
```diff
@@ -288,10 +285,10 @@ static bool FIRCLSMachExceptionRegister(FIRCLSMachExceptionReadContext* context,
...
 
   // Get the desired mask, which covers all the mach exceptions we are capable of handling,
-  // but clear out any that are in our ignore list.  We do this by ANDing with the bitwise
-  // negation.  Because we are only clearing bits, there's no way to set an incorrect mask
-  // using ignoreMask.
-  context->mask = FIRCLSMachExceptionMask() & ~ignoreMask;
+    // but clear out any that are in our ignore list.  We do this by ANDing with the bitwise
+    // negation.  Because we are only clearing bits, there's no way to set an incorrect mask
+    // using ignoreMask.
+    context->mask = FIRCLSMachExceptionMask();
```
Comment (samedson) [inline Crashlytics/Crashlytics/Handlers/FIRCLSMachException.c:291]: Weird indentation?

Comment (elenadoty) [reply Crashlytics/Crashlytics/Handlers/FIRCLSMachException.c]: yep not sure what happened there

**Review by samedson — APPROVED** [review]

**Pull request merged**